code/game/objects/objs.dm

```diff
@@ -44,25 +41,34 @@
 		else
 			T.add_blueprints_preround(src)
 
-/obj/Topic(href, href_list, var/nowindow = 0, var/datum/topic_state/state = GLOB.default_state)
+/obj/Initialize(mapload)
+	. = ..()
+	if (islist(armor))
```

Comment:
spacing.

we don't of `If (` `while (` `for (` or `else if (`

We put the parenthesis right next to things.

Reply:
Gah, it's a habit of mine and I just can't seem to shake it off on DM, thanks!